doc/src/modules/physics/vector/advanced.rst

```diff
@@ -198,3 +200,44 @@ so dynamic symbols created before or after will print the same way.
 Also note that ``Vector``'s ``.dt`` method uses the ``._t`` attribute of
 ``dynamicsymbols``, along with a number of other important functions and
 methods. Don't mix and match symbols representing time.
+
+
+
+Solving Vector Equations
+---------------
+Solving for unknown scalar variables in a set of vector equations is a common need, but 
+SymPy's solvers do not work directly with the physics vectors, so you must first transform
+the vector equations into a set of scalar equations. The `to_matrix()` method provides
```

Comment:
Can you use a reference for `to_matrix`, i.e.
```
:func:`sympy.physics.vector.vector.Vector.to_matrix`
```
or
```
:func:`to_matrix()<sympy.physics.vector.vector.Vector.to_matrix>`
```
Just check what renders well and if the link is working.